app/code/Magento/Checkout/Controller/Sidebar/UpdateItemQty.php

```diff
@@ -55,7 +55,7 @@ public function __construct(
     public function execute()
     {
         $itemId = (int)$this->getRequest()->getParam('item_id');
-        $itemQty = (int)$this->getRequest()->getParam('item_qty');
```

Comment:
This file should not be modified. Please leave it as it was originally. I mentioned it in my previous comment.

Reply:
Done